packages/php-db-import-export/tests/functional/Snowflake/ToFinal/SqlBuilderTest.php

```diff
@@ -1055,6 +1055,97 @@ public function testGetUpdateWithPkCommandConvertValuesWithTimestamp(): void
             );
         }
     }
+    public function testGetUpdateWithPkCommandNullManipulationWithTimestamp(): void
+    {
+        $timestampInit = new DateTime('2020-01-01 00:00:01');
+        $timestampSet = new DateTime('2020-01-01 01:01:01');
+        $this->createTestSchema();
+        $this->createTestTableWithColumns(true);
+        $this->createStagingTableWithData(false);
+
+        // create fake destination and say that there is pk on col1
+        $fakeDestination = new SnowflakeTableDefinition(
+            self::TEST_SCHEMA,
+            self::TEST_TABLE,
+            true,
+            new ColumnCollection([
+                $this->createNullableGenericColumn('col1'),
+                $this->createNullableGenericColumn('col2'),
+            ]),
+            ['col1']
+        );
+        // create fake stage and say that there is less columns
```

Comment:
nechapem uplne ten komentar `less columns`, ma to rovnaky pocet stlpcov ale nema nastaveny PK

Reply:
To bude asi protože createStagingTableWithData dělá tabulku kde je pk1,pk2,col1,col2